<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="CunstomizedControl" name="myWidget" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>211</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="getValue">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>获取当前值</string>
   </property>
  </widget>
  <widget class="QPushButton" name="set2Half">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>设置到50%</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CunstomizedControl</class>
   <extends>QWidget</extends>
   <header location="global">cunstomizedcontrol.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
